optuna/importance/_ped_anova/evaluator.py

```diff
@@ -213,13 +251,18 @@ def evaluate(
             param_importances.update({k: 0.0 for k in single_dists})
             return {k: 0.0 for k in param_importances}
 
-        top_trials = self._get_top_trials(study, trials, params, target)
-        quantile = len(top_trials) / len(trials)
+        target_trials = self._get_top_quantile_trials(study, trials, self._target_quantile, target)
+        region_trials = (
+            trials
+            if self._region_quantile == 1.0
+            else self._get_top_quantile_trials(study, trials, self._region_quantile, target)
+        )
+        quantile = len(target_trials) / len(region_trials)
```

Comment:
```suggestion
        if len(target_trials) == len(region_trials):
            _logger.warning(
                "Target and region quantiles select the same set of trials. "
                "Parameter importances will all be equal."
            )
        quantile = len(target_trials) / len(region_trials)
```

Even if `target_quantile` < `region_quantile` holds, it is still possible that `len(target_trials) == len(region_trials)` when the two quantiles are very close, especially with a small number of trials. So how about implementing the warning, which used to be available in `_get_top_trials`?

Reply:
Thank you for the comment! I'll incorporate this suggestion